### PR TITLE
[doc] Add example and notice about task type Dependent

### LINKED ISSUE #8018: [Feature][Document] Add example and notice about task type Dependent
Due to our tasks task description is not containing some detail, and I think we should add a new section notice and give an example to the task dependent in the document https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/task/dependent.html

### PR DESCRIPTION
**Pull request by QuakeWang**
close: #8018

### CODE REVIEW

**Comment by QuakeWang** [comment]
@zhongjiajie PTAL

File: docs/docs/en/guide/task/dependent.md [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
-> If the weekly report A also needs to be executed successfully last Tuesday:
+## Examples
 
- <p align="center">
-   <img src="/img/depend-node3-en.png" width="80%" />
- </p>
\ No newline at end of file
+The Dependent node provides a logical judgment function, which can detect the execution of the dependent node according to the logic.
+
+For example, process A is a weekly report task, processes B and C are daily tasks, and task A requires tasks B and C to be successfully executed every day of the last week, as shown in the figure:
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/dependent.md]:
```suggestion
For example, process A is a weekly task, processes B and C are daily tasks, and task A requires tasks B and C to be successfully executed every day of the last week.
```

File: docs/docs/en/guide/task/dependent.md [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
+
+![dependent_task01](/img/tasks/demo/dependent_task01.png)
+
+For example, process A is a weekly report task, processes B and C are daily tasks, and task A requires tasks B or C to be successfully executed every day of the last week, as shown in the figure:
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/dependent.md]:
```suggestion
And another example is that process A is a weekly report task, processes B and C are daily tasks, and task A requires tasks B or C to be successfully executed every day of the last week:
```

**Review by zhongjiajie — DISMISSED** [review]
LGTM, thanks @QuakeWang

**Review by zhongjiajie — APPROVED** [review]
LGTM now

**Pull request merged**